cli/tests/integration_tests.rs

```diff
@@ -320,6 +320,12 @@ itest!(_036_import_map_fetch {
   output: "036_import_map_fetch.out",
 });
 
+itest!(_037_fetch_multiple {
+  args: "fetch --reload fetch/test.ts fetch/other.ts",
+  check_stderr: true,
+  output: "037_fetch_multiple.out",
+});
+
```

Comment:
Great, but unfortunately this test now relies on `unpkg.com` - could you change it to change import some other files present in `tests/` directory?

Reply:
localhost:4545/cli/tests/hello.txt

Reply:
Sure, will do now!

Reply:
Done!